src/main/java/com/thebois/models/beings/Pawn.java

```diff
@@ -1,10 +1,15 @@
 package com.thebois.models.beings;
 
 import java.util.Collection;
+import java.util.Optional;
 import java.util.Random;
 
+import com.thebois.models.IFinder;
 import com.thebois.models.Position;
 import com.thebois.models.beings.pathfinding.IPathFinder;
+import com.thebois.models.inventory.items.Log;
+import com.thebois.models.inventory.items.Rock;
```

Comment:
I know, but they cant collect material yet, so it is only a temporary solution

Reply:
I wondered why it used these items, but the non line 85-86 you can see that they're used to deliver items to structures. This is fine for now, since I'll refactor it to be in a BuildTask later, so Pawn will not know about it then.

But if this branch has been merged with master, Log and Rock shouldn't exist anymore, you should have to use the factory to create the items (which means a looser dependency on just IItem).